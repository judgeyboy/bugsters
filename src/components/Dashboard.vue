<template>
  <div class="dashboard">
    <h2>DASHBOARD</h2>
    <div class="columns">
      <div class="column col-sm-12 col-6">
        <Orders />
      </div>
    </div>
  </div>
</template>

<script>
import Orders from './Orders'

export default {
  name: 'Dashboard',
  components: {
    Orders
  }
}
</script>
