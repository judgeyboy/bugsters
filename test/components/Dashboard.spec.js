import {shallow} from 'vue-test-utils'

import Dashboard from '@/components/Dashboard'

describe('Dashboard component', () => {
  it('should be a vue instance', () => {
    const wrapper = shallow(Dashboard)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(Dashboard)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
