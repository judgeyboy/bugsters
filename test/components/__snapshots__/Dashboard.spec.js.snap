// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component should match snapshot 1`] = `"<div class=\\"dashboard\\"><h2>DASHBOARD</h2> <div class=\\"columns\\"><div class=\\"column col-sm-12 col-6\\"><!----></div></div></div>"`;
